cobbler/actions/sync.py

```diff
@@ -18,6 +18,8 @@
 from cobbler import tftpgen
 from cobbler import utils
 from cobbler.utils import filesystem_helpers
+from cobbler.items.system import System
+from cobbler.items.profile import Profile
```

Comment:
This neat little trick enables us to prevent cyclic imports. (see: https://docs.python.org/3.6/library/typing.html#typing.TYPE_CHECKING)

```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from cobbler.items.system import System
    from cobbler.items.profile import Profile
```

Reply:
Okay. I will try to check the whole file again. In theory we can also only try to add the import conditionally for Profile but I am not aware that we actually should need it in `sync.py`.